src/twin/network_status.rs

```diff
@@ -188,17 +188,18 @@ impl NetworkStatus {
                     name: i.name.clone(),
                 });
 
-                match i.addr {
-                    Some(Addr::V4(addr)) => entry
-                        .addr_v4
-                        .get_or_insert(vec![])
-                        .push(addr.ip.to_string()),
-                    Some(Addr::V6(addr)) => entry
-                        .addr_v6
-                        .get_or_insert(vec![])
-                        .push(addr.ip.to_string()),
-                    None => error!("report_network_status: ip address is missing"),
-                };
+                for a in &i.addr {
```

Comment:
i guess that's a change which has nothing to do with the dependency updates and is a fix for publishing all ip's. imho should be reflected in PR description.

Reply:
no, it is a result of the update